<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpgradeStationServiceDlg</class>
 <widget class="QDialog" name="UpgradeStationServiceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>升级中控</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>选择中控服务文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="sendFileNamesPlainTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browserPushButton">
       <property name="text">
        <string>选择文件...</string>
       </property>
       <property name="icon">
        <iconset resource="cc-client.qrc">
         <normaloff>:/Icons/search.ico</normaloff>:/Icons/search.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upgradePushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>开始升级</string>
       </property>
       <property name="icon">
        <iconset resource="cc-client.qrc">
         <normaloff>:/Icons/52design.com_jingying_059.png</normaloff>:/Icons/52design.com_jingying_059.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="versionLabel">
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>674</width>
        <height>309</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cc-client.qrc"/>
 </resources>
 <connections/>
</ui>
